Universal/brg0.md

```diff
@@ -0,0 +1,16 @@
+# Fixing Property Injection (SSDT-BRG0)
+
+* [Fixing Property Injection (SSDT-BRG0)](#fixing-property-injection-ssdt-brg0)
+  * [What this SSDT does](#what-this-ssdt-does)
+  * [Methods to make this SSDT](#methods-to-make-this-ssdt)
+
+## What this SSDT does
+
+On some platforms, the PCI bridge your GPU is connected to is not defined.
```

Comment:
Maybe explain (quickly) why this causes problems. Iirc it's to do with device properties not injecting right?